src/server/db_slice.h

```diff
@@ -598,6 +601,7 @@ class DbSlice {
   size_t soft_budget_limit_ = 0;
   size_t table_memory_ = 0;
   uint64_t entries_count_ = 0;
+  unsigned load_ref_count_ = 0;
```

Comment:
I think we should add a dcheck that this ref count == 0 when we change server state to active
in Service::SwitchState